Plots/Masking/NCL_mask_5.py

```diff
@@ -0,0 +1,154 @@
+"""
+NCL_mask_5.py
+================
+
+This script illustrates the following concepts:
+    - Using cartopy.feature options to display map information
+    - Paneling two plots on a page with 'subplots' command
+    - Drawing contours
+    - Explicitly setting the fill colors for contours
+    - Drawing contours over land only
+    - Using draw order resources to mask areas in a plot
+    - Adding a label bar
+    - Implementing best practices when choosing a color scheme 
+
+    
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/mask_5.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/mask_5_lg.png
+                         
+"""
+###############################################################################
+# Import packages:
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+import matplotlib.patches as mpatches
+
+from geocat.viz import util as gvutil
+import geocat.datafiles as gdf
+###############################################################################
+# Plot
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/atmos.nc"), decode_times=False)
+t = ds.TS.isel(time=0)
+
+# Fix the artifact of not-shown-data around 0 and 360-degree longitudes
+wrap_t = gvutil.xr_add_cyclic_longitudes(t, "lon")
+                    
+###############################################################################
+#Plot:
+
+fig = plt.figure(figsize=(12, 14))
+
+def Plot(row, col, pos, title):
```

Comment:
If you use my suggestion above, you could then pass in the `axes` instance instead of `row`, `col`, and `pos`. This will also make your helper functions more intuitive.